### PR TITLE
chore(demo-gatsby): media updates

### PR DESCRIPTION
**Pull request by kendallstrautman**
Updates the demo to work with latest media changes. I had to adjust the way the selected media preview src was being generated with wysiwyg images...I'm not sure if this is the best way to handle the `previewSrc`. The previous code didn't account for promises. 

Also, a small style fix in the media manager.

### CODE REVIEW

**Comment by kendallstrautman** [comment]
@ncphillips just needed the extra .3rem 😅

**Comment by kendallstrautman** [comment]
@all-contributors please add @kud for docs

File: packages/demo-gatsby/src/templates/blog-post.js [hunk 1]
```diff
@@ -189,24 +189,22 @@ function BlogPostTemplate(props) {
...
 
-                  let media = await cms.media.store.persist(
-                    files.map(file => ({
-                      directory,
-                      file,
-                    }))
-                  )
+                // Decide the file upload directory for the post
+                uploadDir: blogPost => {
+                  const postPathParts = blogPost.fileRelativePath.split("/")
```
Comment (dwalkr) [inline packages/demo-gatsby/src/templates/blog-post.js:197]: could replace this whole function w/ a call to `path.dirname` https://nodejs.org/api/path.html#path_path_dirname_path

File: packages/demo-gatsby/src/templates/blog-post.js [hunk 2]
```diff
@@ -189,24 +189,22 @@ function BlogPostTemplate(props) {
...
+                  return postDirectory
                 },
-                previewUrl(src) {
-                  return new Promise(resolve => {
-                    setTimeout(() => {
-                      resolve(src)
-                    }, 3000)
-                  })
+                previewSrc(src) {
+                  const formattedSrc = src.replace(/\/public|\/static/, "")
```
Comment (dwalkr) [inline packages/demo-gatsby/src/templates/blog-post.js:206]:
```suggestion
                  const formattedSrc = src.replace(/^\/public|\/static/, "")
```

**Review by dwalkr — CHANGES_REQUESTED** [review]

**Review by dwalkr — APPROVED** [review]

**Pull request merged**